pages/_app.js

```diff
@@ -58,7 +58,7 @@ class MyApp extends App {
           config={{
             apiKey: API_KEY,
             host: host,
-            forceRedirect: true,
+            forceRedirect: false,
```

Comment:
Can you explain what this change does?

Reply:
The "infinite redirect" was happening because the "shopify library" we use was automatically redirecting the user to the shopify app

Before it was working because our "redirect" was happening before.

Now with the calls we're making (i.e, get tenants and get linked sources), the "redirect" from the "shopify library" was being triggered...

This `true` -> `false` change disables this "shopify library redirect"